### ISSUE TITLE
Did you update the moving averages when taking the batch norm ? 

### ISSUE THREAD

**Issue by shamanez** [yscacaca/DeepSense#16]
In this [line ](https://github.com/yscacaca/DeepSense/blob/master/deepSense_HHAR_tf.py#L74). It even says ,

**updates_collections=None**

What is actually meant by this ?

**Comment by shamanez** [comment]
The documentation in tensorflow say like this , 

> Note: when training, the moving_mean and moving_variance need to be updated. By default the update ops are placed in tf.GraphKeys.UPDATE_OPS, so they need to be added as a dependency to the train_op. For example:

**Comment by yscacaca** [comment]
updates_collections=None is to force the updates in place. You can also tf.GraphKeys.UPDATE_OPS by referring the TensorFlow documents.
